samples/MvcSample.Web/project.json

```diff
@@ -1,6 +1,6 @@
 {
   "commands": {
-    "web": "Microsoft.AspNet.Hosting server=Microsoft.AspNet.Server.WebListener server.urls=http://localhost:5001",
```

Comment:
This was a change automatically made by VS(i left it thinking that it is anyways correct...as one should be able to do Ctrl + F5 on a sample in VS)...but this is now not required due to the sand box changes...